packages/wallet/frontend/src/components/WalletAddressesTable.tsx

```diff
@@ -101,7 +102,7 @@ export const CopyWalletAddress = () => {
   const { walletAddress } = useContext(WalletAddressRowContext)
   const [isCopied, setIsCopied] = useState(false)
 
-  let pointer = walletAddress.url
+  let pointer = replaceWalletAddressProtocol(walletAddress.url)
```

Comment:
Here we are already checking if it is the card's wallet address, a few lines below. Remove one of them.